exonum-java-binding/testkit/src/main/java/com/exonum/binding/testkit/TestKit.java

```diff
@@ -352,6 +358,14 @@ public Snapshot getSnapshot() {
     return createSnapshot(snapshotCleaner);
   }
 
+  /**
+   * Returns a server port of the corresponding runtime, or {@link OptionalInt#empty()} if it
+   * does not currently accept requests.
+   */
+  public OptionalInt getServerPort() {
```

Comment:
I wonder how it can be used. If to test APIs, then the fact that it is not possible to wait until it does accept requests is not good. 
Probably, to support that properly, we will have to somehow integrate VertxExtension with Testkit. 🤔

Reply:
We could remove it for now?

Reply:
Probably yes, because if it returns some port or none depends on if the server has opened the socket.

Reply:
Ok, then I'll remove the ability to specify server port altogether.